pkgs/tools/text/replace/default.nix

```
@@ -4,8 +4,8 @@ stdenv.mkDerivation {
   name = "replace-2.24";
```

```suggestion
  pname = "replace";
  version = "2.24";
```

and add a rec one line above.